variables.tf

```diff
@@ -153,7 +153,7 @@ variable "bucket_enable_ubla" {
 variable "bucket_force_destroy" {
   type        = bool
   default     = true
-  description = "Force destroy bucket (Required when bucket not empty)"
+  description = "Force destroy bucket (if disabled, terraform will not be able do destroy non-empty bucket)"
```

Comment:
nit:do -> to

`if disabled, terraform will not be able do destroy non-empty bucket` -> `if disabled, terraform will not be able to destroy non-empty bucket`